CONTRIBUTING.md

```diff
@@ -400,4 +398,4 @@ git config --global core.autocrlf true
 
 After you run this command, you should be able to checkout projects that are maintained using CRLF (Windows) again.
 
-[^1]: <https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf>
+[^1]: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf
```

Comment:
@jonaro00 do you recall why you changed this? Removing it fixes the parsing error in the docs, but it might be needed?

Reply:
Mainly due to markdown lint rules, but as you also have discovered, Mintlify's markdown parser does not even support comments...

Reply:
In my other large docs PR, using Mintlify v3, I changed the Contributing sidebar item to be a link to CONTRIBUTING.md instead. Sacrificing the quality (graph etc) from GitHub's rendering to mirror it is a waste IMO. A contributor should not be afraid of starting out on a markdown file on GH.

Reply:
Yes, that is a very fair point, there certainly are some footguns, I encountered a few myself testing this. Our understanding was that they would support mermaid soon, but they must not have had time to integrate it yet. :sweat_smile: 

I will merge this for now so we fix the parsing error, but I think just linking to github is definitely worth considering. I'll ask Ivan if they replied about mermaid as well.